File: QE-selenium/java/Activity9.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity9 {

    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();
        Actions actions = new Actions(driver);

        driver.get("https://training-support.net/webelements/keyboard-events");
        System.out.println(driver.getTitle());

        WebElement textInput = driver.findElement(By.xpath("//h1[@class='mt-3 text-center text-4xl font-semibold text-black']"));

        actions.click(textInput).sendKeys("Karthik").perform();

        System.out.println(textInput.getText());

        driver.quit();
    }
}

File: QE-selenium/python/Activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys
from selenium.webdriver.common.action_chains import ActionChains

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/keyboard-events")
    print(driver.title)
    text_input = driver.find_element(By.XPATH,"//h1[@class='mt-3 text-center text-4xl font-semibold text-black']")
    ActionChains(driver).click(text_input).send_keys("Karthik").perform()
    print(text_input.text)
    driver.quit()
